package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "build": "tsc",
     "dev": "tsc --watch",
-    "publishToNPM": "yarn version --patch && yarn build && yarn publish --access public",
+    "publishToNPM": "yarn build && yarn publish --access public",
```

Comment:
Do we no longer want to run `yarn version --patch` before we publish?

Reply:
I think yarn publish does this automatically for us anyway.